### ISSUE TITLE
JAWS 2025 does not announce the ariavaluetext of a slider

### ISSUE THREAD

**Issue by roynickelson** [FreedomScientific/VFO-standards-support#881]
### Summary
JAWS 2025 does not announce the ariavaluetext of a slider
> Brief description of the issue. Please list any specific steps.
> 
1. Navigate to https://www.w3.org/WAI/ARIA/apg/patterns/slider/examples/slider-seek/
2: navigate to the slider and turn on forms mode. Navigate to 91. What should happen: JAWS should announce 1 minute 31 seconds  What happens instead JAWS only announces 91.  This works properly with JAWS 2024.

#### JAWS version and build number
JAWS 2025.2412.50.

 
#### Operating System and version
Windows 11 24h2

#### Browser and version:
Google Chrome 131.0.6778.265.
